docs/home_assistant.md

```diff
@@ -6,15 +6,15 @@ layout: default
 Home Assistant Component
 ========================
 
-XKNX ships with [Home Assistant](https://home-assistant.io/components/#search/knx), the great platform for home automation!
+XKNX is shipped within [Home Assistant](https://www.home-assistant.io/), the great platform for home automation, in the form of the included [KNX component](https://www.home-assistant.io/integrations/#search/KNX).
 
-For development and debugging reasons - or to catch up with the newest features - you may still use the custom component version of the plugin.
+For development and debugging reasons - or to catch up with the newest features - you may still want to use the custom component version of the plugin.
```

Comment:
Maybe we shouldn't call it plug-in as it could be easily confused with "add-on" which is a different Home-Assistant concept. 
I'd say integration or platform.